guides/common/assembly_importing-content.adoc

```diff
@@ -36,6 +36,8 @@ include::modules/proc_changing-the-download-policy-for-a-repository.adoc[levelof
 
 include::modules/proc_uploading-content-to-custom-rpm-repositories.adoc[leveloffset=+1]
 
+include::modules/proc_configuring-selinux-to-permit-access-to-repos-on-custom-ports.adoc[leveloffset=+1]
```

Comment:
This is what should be shielded by `ifdef`.
```suggestion
ifdef::katello,satellite,orcharhino[]
include::modules/proc_configuring-selinux-to-permit-access-to-repos-on-custom-ports.adoc[leveloffset=+1]
endif::[]
```

Reply:
Change made